File: num3.m
clc; clear all; close all;
pkg load database;
fprintf('Bienvenido al programa\n');
fprintf('A continuacion ingrese 3 numeros enteros positivos\n');
a=0;
b='';
n1=input('Primer Numero: ');
n2=input('Segundo Numero: ');
n3=input('Tercero Numero: ');

if n1>n2 && n1>n3
  fprintf('El primer numero es el mas grande\n');
  a=n1+n2+n3;
  b=mat2str(a);
endif

if n2>n1 && n2>n3
  fprintf('El segundo numero es el mas grande\n');
  a=n1*n2*n3;
  b=mat2str(a);
endif

if n3>n1 && n3>n2
  fprintf('El tercer numero es el mas grande\n');
  t1=mat2str(n1);
  t2=mat2str(n2);
  t3=mat2str(n3);
  b= strcat(t1, t2, t3);
endif

if n1==n3 && n1==n2 && n2==n3
  fprintf('Todos son iguales: %d, %d, %d \n', n1, n2, n3);
  b='Todos son iguales';
endif
fprintf('%s \n', b);

params= cell(1,4);
params{1,1}=n1;
params{1,2}=n2;
params{1,3}=n3;
params{1,4}=b;
save('num3.txt','b','-ascii');
conn = pq_connect(setdbopts('dbname','HWP','host','localhost',
'port','5432','user','postgres','password','123fgthg'));
N=pq_exec_params(conn, "insert into ac3(n1, n2, n3, res) values($1,$2,$3,$4);",params); %insertar datos en la tabla

File: num3.py
import psycopg2
def  post(x,y,z,w):
    try:
        conexion = psycopg2.connect(
            host = "localhost",
            port = "5432",
            user = "postgres",
            password = "123fgthg",
            dbname = "HWP"
            )
    except:
        print("Sin Conexion Exitosa\n")
            
    cursor = conexion.cursor()
    cursor.execute("INSERT INTO ac3(n1, n2, n3, res) VALUES (%s, %s, %s, %s);", (x, y, z, w))
    conexion.commit()
    cursor.close()
    conexion.close()
print("A continuacion ingrese 3 numeros")
n1 = int(input("Primer Numero \n"))
n2 = int(input("Segundo Numero \n"))
n3 = int(input("Tercer Numero \n"))
a = ""
if n1>n2 and n2>n3:
    print("El Primer numero es el mas grande\n")
    t = n1+n2+n3
    a = str(t)
    print(a)
elif n2>n1 and n2>n3:
    print("El Segundo numero es el mas grande\n")
    t = n1*n2*n3
    a = str(t)
    print(a)
elif n3>n1 and n3>n2:
    print("El Tercer numero es el mas grande\n")
    a = str(n1)+str(n2)+str(n3)
    print(a)
elif n1==n2==n3:
    print("Todos son iguales:"+ str(n1)+", "+str(n2)+", "+str(n3))
    a="TODOS SON IGUALES"

post(n1,n2,n3,a)